.github/workflows/base-docker-image.yml

```diff
@@ -0,0 +1,22 @@
+name: "Generate apply-base docker image"
+on:
```

Comment:
you could test this by changing to `on: push` temporarily

Reply:
I'll test with a manual start after merging to main